keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1918,6 +1932,13 @@ currently declared on the Pod and Device Plugin level.
 Extending the Device Plugins API to use [Container Device Interface](https://github.com/container-orchestrated-devices/container-device-interface)
 would help address some of the requirements, but not all of them.
 
+NodePrepareResource and NodeUnprepareResource could be added as optional
+calls to the Device Plugins API. However, this would mean that
+developers of the Device Plugins would have to implement mandatory
+API calls (ListAndWatch, Allocate), which could create confusion
+as those calls are meaningless for the Dynamic Resource Allocation
+purposes.
```

Comment:
```suggestion
purposes.

Even worse, existing device plugins would have to implement the new
calls with stubs that return errors because the generated Go interface
will require them.
```